src/picknik_ur_site_config/objectives/open_door_fixed_handle_ml.xml

```diff
@@ -0,0 +1,58 @@
+      <!-- Update the admittance controller and reset the FTS -->
+      <SubTree ID="Re-Zero Force-Torque Sensors" _collapsed="true"/>
+      <SubTree ID="Update Admittance Controller" _collapsed="true"/>
+      <!-- Look at the desired location where objects (door) will be present -->
+      <Action ID="MoveToWaypoint" waypoint_name="Extended Right" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="true" constraints=""/>
+      <SubTree ID="Clear Snapshot" _collapsed="true"/>
+      <SubTree ID="Reset Planning Scene" _collapsed="true"/>
+      <SubTree ID="GetDoorGraspableObject" _collapsed="true" door_object="{door_object}" detection_classes="space booth cabinet door"/>
+      <!-- Get the masks for the handle subframe of the door GraspableObject -->
+      <!-- May require multiple view-points due to incomplete point clouds due to angle. -->
+      <Control ID="Fallback">
+        <SubTree ID="GetFixedHandleGraspableObject" _collapsed="true" grasp_pose="{grasp_pose}"/>
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Extended Right 2" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="true" constraints=""/>
+          <SubTree ID="GetFixedHandleGraspableObject" _collapsed="true" grasp_pose="{grasp_pose}"/>
+        </Control>
+        <Control ID="Sequence">
+          <Action ID="MoveToWaypoint" waypoint_name="Extended Right" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="true" constraints=""/>
+          <SubTree ID="GetFixedHandleGraspableObject" _collapsed="true" grasp_pose="{grasp_pose}"/>
+        </Control>
+      </Control>
+      <!-- Get the masks for the hinge subframes of the door GraspableObject -->
+      <Action ID="MoveToWaypoint" waypoint_name="Hinge View" planning_group_name="manipulator" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" use_all_planners="false" constraints=""/>
+      <SubTree ID="GetHingeLineSegment" _collapsed="true" line_segment_start_point="{line_segment_start_point}" line_segment_end_point="{line_segment_end_point}"/>
+      <!-- Helper behavior to convert poses into subframes that adhere to the expected axis convention -->
+      <Action ID="GetMoveAlongArcSubframes" target_grasp_pose="{grasp_pose}" hinge_axis_pose_start="{line_segment_start_point}" hinge_axis_pose_end="{line_segment_end_point}" move_along_arc_subframes="{move_along_arc_subframes}"/>
+      <!-- Add the subframes to the Object-->
+      <Decorator ID="ForEachObjectSubframe" vector_in="{move_along_arc_subframes}" out="{subframe}">
+        <Control ID="Sequence">
+          <Action ID="AddSubframeToObject" subframe="{subframe}" graspable_object="{door_object}"/>
+        </Control>
```

Comment:
Found another simplifying thing: Since this is just one node, you can remove the Sequence and connect the AddSubframeToObject directly to the parent decorator.

Same holds for other places in this PR where this pattern repeats